axiom/nr_distributed_trace.h

```diff
@@ -12,6 +12,9 @@
 #include "util_time.h"
 #include "util_object.h"
 
+/* This is the maximum size of trace id that the backend will accept */
+#define NR_TRACE_ID_SIZE 32
```

Comment:
Better name would be `NR_TRACE_ID_MAX_SIZE` since we know the nr trace id can also be 16.

Reply:
Addressed in f58c3f9b05fec9ffcc9672e5759b4f76fb926725.